CMakeLists.txt

```diff
@@ -20,163 +20,34 @@ endif()
 INCLUDE(CMakeDependentOption)
 INCLUDE(CMakeParseArguments)
 
-# Find the modules included with Xcompact
-#SET(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
-
 # make sure that the default is a RELEASE
 if (NOT CMAKE_BUILD_TYPE)
   set (CMAKE_BUILD_TYPE RELEASE CACHE STRING
-      "Choose the type of build, options are: None Debug Release."
+      "Choose the type of build, options are: None Dev Debug Release."
```

Comment:
Are we keeping both dev and debug builds ?

Reply:
They can be useful: Debug just enables runtime checks, Dev additionally makes the compiler more strict - however users may not want this strictness so I think it is useful to have the option to select between these.

Reply:
Then the CMake variable `X3D_FFLAGS_DEV` should be defined for each compiler ? Currently it is not defined for Cray, Fujitsu and Nag.

Reply:
I'll add the flag also for Cray and Fujitsu. Do we need Nag?

Reply:
no

Reply:
Ok I'll remove it